tests/testthat/test-verb-joins.R

```diff
@@ -72,6 +72,42 @@ test_that("join works with in_schema", {
   )
 })
 
+test_that("alias truncates long table names at database limit", {
+  # Postgres has max table name length of 63; ensure it's not exceeded when generating table alias
+  withr::local_db_connection(con <- dbConnect(RSQLite::SQLite(), ":memory:"))
```

Comment:
```suggestion
  con <- withr::local_db_connection(dbConnect(RSQLite::SQLite(), ":memory:"))
```

Reply:
This was carried forward from `join works with in_schema` test. Not applicable now using Postgres backend, so will revert